app/config/message-queue.js

```diff
@@ -25,8 +25,8 @@ const sharedConfig = {
 }
 
 const config = {
-  applicationdDocCreationRequestQueue: {
-    address: process.env.APPLICATIONDOCCREATIONREQUEST_QUEUE_ADDRESS,
+  certificateRequestQueue: {
+    address: process.env.CERTIFICATE_REQUEST_QUEUE_ADDRESS,
```

Comment:
I'd be inclined to use CERIFICATE_REQUEST_SERVICE_BUS_ADDRESS instead of QUEUE to avoid confusion between SB and Storage Queues - but very minor. Also applies to config-map etc